### PR TITLE
Added ask path feature for creating new project

### PR DESCRIPTION
**Pull request by SudeepPatel-0812**
Added the functionality to add path instead of creating path in the current directory if the path is not defined the new project will be made in the current directory.

### CODE REVIEW

File: utils/createStrapi.js [hunk 1]
```diff
@@ -27,12 +28,59 @@ const createStrapiProject = async () => {
 				active: `Yes`,
 				inactive: `No`,
 				type: `toggle`
+			},
+			{
+				name: `projectPath`,
```
Comment (Eventyret) [inline utils/createStrapi.js:33]: We can give this an intial value of the current `process.cwf()` I think 🤔 that way they don't need to type it all the time.

File: index.js [hunk 2]
```diff
@@ -29,26 +29,33 @@ const {
...
-			await createStrapiProject();
+			projectPath = await createStrapiProject();
 		}
 
 		if (input.includes(`reset`)) {
 			await reset();
 			goodbye();
 			return;
 		}
+		process.chdir(projectPath);
```
Comment (Eventyret) [inline index.js:57]: if we are only using this in `detecthStrapiProject()` can't we then do something like this ?

```javascript
if (!(await detectStrapiProject())) {
			const projectPath = await createStrapiProject();
process.chdir(projectPath);
setConfig({outDir: path.join(process.cwd())});
}
```
Not sure but I think we can ?

**Review by Eventyret — CHANGES_REQUESTED** [review]

File: utils/createStrapi.js (hunk 1 above)
Comment (SudeepPatel-0812) [reply utils/createStrapi.js:33]: made the change now the initial value has process.cwd()

File: index.js (hunk 2 above)
Comment (SudeepPatel-0812) [reply index.js]: made the change and moved the code inside if condition

**Review by Eventyret — APPROVED** [review]

**Comment by Eventyret** [comment]
@SudeepPatel-0812 this should be ready to be merged. if you can't merge it let me know and I will merge it for you :)

**Pull request merged**